packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -129,7 +129,7 @@ function setupTestSuiteMatrix(
             // disconnect might also fail, so ignoring the error here
           })
           if (clientMeta.dataProxy) {
-            await stopMiniProxyQueryEngine(client)
+            await stopMiniProxyQueryEngine(client, globalThis['datasourceInfo'])
```

Comment:
I did some changes to improve miniproxy startup times, detected because I had some failing tests.